flask_restalchemy/resources/utils.py

```diff
@@ -131,6 +125,47 @@ def build_filter_operator(column_name, request_filter, serializer):
     if 'limit' in request.args:
         limit = request.args['limit']
         query = query.limit(limit)
+
+    return query
+
+
+def query_from_request(model, model_serializer, request, query=None):
+    """
+    Perform a filtered search in the database model table using query parameters in the http URL,
+    disposed on the request args. The default logical operator is AND, but you can set the OR as
+    in the following examples:
+
+        a) OR -> ?filter={"$or":{"name": {"startswith": "Terrans 1"},"location": "Location 1"}}
```

Comment:
I think you could ref `apply_request_args_to_query` docs to avoid duplicate these. Use Sphinx syntax : `see :func:apply_request_args_to_query on how to use filters`